config/content.d/staging.horse.json

```diff
@@ -5,6 +5,7 @@
       "/docs/private-cloud/release-notes/": "https://github.com/rcbops/rpc-openstack/master/",
       "/docs/api-doc-template/": "https://github.com/rackerlabs/docs-common/",
       "/docs/billing-api/": "https://github.com/rackerlabs/docs-billing-apiguide/"
+      "/docs/managed-kubernetes/master/": "https://github.com/rcbops/kubernetes-installer/master"
```

Comment:
Might need / after master on this line. Does the colon show up as red for you? Just curious...

Reply:
fixed.